tests/provision_test.go

```diff
@@ -91,6 +93,21 @@ func blockVolCreationTest() {
 	By("Deleting storage class", deleteStorageClass)
 }
 
+func sharedVolumeTest() {
+	By("Creating shared LV storage class", createSharedVolStorageClass)
+	By("creating and verifying PVC bound status", createAndVerifyPVC)
+	//we use two fio app pods for this test.
+	appNames = append(appNames, "fio-ci-1")
```

Comment:
Could we add a new constant appNames?

Reply:
As discussed, since all deploy and verify functions use the same const, it made more sense to reuse the same.